MAC/on_appliance/map-source.sh

```diff
@@ -58,4 +58,19 @@ echo "**** Run: git update-index --assume-unchanged log/.gitkeep on MAC."
 [[ -d $MIQ_DIR/public/assets && ! -L $MIQ_DIR/public/assets ]] && rm -rf $MIQ_DIR/public/assets
 ln -f -s $LOCAL_ASSETS_DIR $MIQ_DIR/public/assets
 
+# We don't want compiled assets written to our source tree.
+[[ -d $LOCAL_VENDOR_ASSETS_DIR ]] || mkdir -p $LOCAL_VENDOR_ASSETS_DIR
+[[ -d $MIQ_DIR/vendor/assets && ! -L $MIQ_DIR/vendor/assets ]] && rm -rf $MIQ_DIR/vendor/assets
+ln -f -s $LOCAL_VENDOR_ASSETS_DIR $MIQ_DIR/vendor/assets
+
+# Allow Apache to follow the symbolic link created above and set selinux accordingly
```

Comment:
@roliveri done as suggested.  Note that we also have to source in the defines.sh definitions.